<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompetitionManager</class>
 <widget class="QMainWindow" name="CompetitionManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompetitionManager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icons/cm_icon.png</normaloff>:/icons/cm_icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sideBar" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #404040</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="Wszystkie">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border: none;
	color: white
}
QToolButton:checked{
	background-color: #262626;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/resources.qrc">
           <normaloff>:/icons/all.png</normaloff>:/icons/all.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SiatkowkaPlazowa">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border: none;
	color: white
}
QToolButton:checked{
	background-color: #262626;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/resources.qrc">
           <normaloff>:/icons/ball.png</normaloff>:/icons/ball.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="PrzeciaganieLiny">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border: none;
	color: white
}
QToolButton:checked{
	background-color: #262626;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/resources.qrc">
           <normaloff>:/icons/rope.png</normaloff>:/icons/rope.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="DwaOgnie">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border: none;
	color: white
}
QToolButton:checked{
	background-color: #262626;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/resources.qrc">
           <normaloff>:/icons/fire.png</normaloff>:/icons/fire.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="tlo" native="true">
      <property name="styleSheet">
       <string notr="true">.QWidget { background-color: #404040; }</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="Karty">
         <property name="styleSheet">
          <string notr="true">QTabWidget::pane { 
	border: 3px solid #bebebe; 
}

QTabBar::tab {
	padding-left: 12px;
	padding-right: 12px;
	padding-top: 4px;
	padding-bottom: 4px;
}

QTabBar::tab:selected {
	color: black;
	background-color: #bebebe;
}

QTabBar::tab:!selected{
	color: white;
	background-color: #353535;
	margin-top: 2px;
}</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="kartaDruzyny">
          <attribute name="title">
           <string>Drużyny</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableView" name="widokDruzyn">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="kartaSedziowie">
          <attribute name="title">
           <string>Sędziowie</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableView" name="widokSedziow">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="kartaMecze">
          <attribute name="title">
           <string>Mecze</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableView" name="widokSpotkan">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar { 
	background-color: #404040;
	color: white; 
}

QMenuBar::item {
	background: transparent;
}

QMenuBar::item:selected {
	color: white;
	background: #353535;
}

QMenuBar::item:pressed {
	background: #303030;
}

QMenu {
	padding: 0px;
	border: 0px;
	background-color: #404040;
	color: white;
	selection-background-color: #262626;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionNowy"/>
    <addaction name="actionWczytaj"/>
    <addaction name="separator"/>
    <addaction name="actionZapisz"/>
    <addaction name="actionZapisz_jako"/>
    <addaction name="separator"/>
    <addaction name="actionZamknij"/>
   </widget>
   <widget class="QMenu" name="menuDruzyny">
    <property name="title">
     <string>Drużyny</string>
    </property>
    <addaction name="actionDodaj_druzyne"/>
    <addaction name="actionEdytuj_druzyne"/>
    <addaction name="actionUsun_druzyne"/>
   </widget>
   <widget class="QMenu" name="menuSedziowie">
    <property name="title">
     <string>Sędziowie</string>
    </property>
    <addaction name="actionDodaj_sedziego"/>
    <addaction name="actionEdytuj_dane_sedziego"/>
    <addaction name="actionUsun_sedziego"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAutomat">
    <property name="title">
     <string>Automat</string>
    </property>
    <addaction name="actionWygeneruj_Druzyny"/>
    <addaction name="actionWygeneruj_Sedziow"/>
    <addaction name="separator"/>
    <addaction name="actionRozegraj_Mecze"/>
   </widget>
   <widget class="QMenu" name="menuZawody">
    <property name="title">
     <string>Zawody</string>
    </property>
    <addaction name="actionStan"/>
    <addaction name="actionZaplanuj_spotkania"/>
    <addaction name="actionNastepny_etap"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDruzyny"/>
   <addaction name="menuSedziowie"/>
   <addaction name="menuAutomat"/>
   <addaction name="menuZawody"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">QStatusBar{
	background-color: #404040;
	color: white;
}

QStatusBar::item{
	border: none;
}
</string>
   </property>
  </widget>
  <action name="actionDodaj_druzyne">
   <property name="text">
    <string>Dodaj drużynę</string>
   </property>
  </action>
  <action name="actionEdytuj_druzyne">
   <property name="text">
    <string>Edytuj drużynę</string>
   </property>
  </action>
  <action name="actionUsun_druzyne">
   <property name="text">
    <string>Usuń drużynę</string>
   </property>
  </action>
  <action name="actionDodaj_sedziego">
   <property name="text">
    <string>Dodaj sędziego</string>
   </property>
  </action>
  <action name="actionEdytuj_dane_sedziego">
   <property name="text">
    <string>Edytuj dane sędziego</string>
   </property>
  </action>
  <action name="actionUsun_sedziego">
   <property name="text">
    <string>Usuń sędziego</string>
   </property>
  </action>
  <action name="actionNowy">
   <property name="text">
    <string>Nowa baza</string>
   </property>
  </action>
  <action name="actionWczytaj">
   <property name="text">
    <string>Wczytaj bazę z pliku</string>
   </property>
  </action>
  <action name="actionZapisz">
   <property name="text">
    <string>Zapisz</string>
   </property>
  </action>
  <action name="actionZapisz_jako">
   <property name="text">
    <string>Zapisz jako</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
  <action name="actionWygeneruj_Druzyny">
   <property name="text">
    <string>Wygeneruj Drużyny</string>
   </property>
  </action>
  <action name="actionWygeneruj_Sedziow">
   <property name="text">
    <string>Wygeneruj Sędziów</string>
   </property>
  </action>
  <action name="actionRozegraj_Mecze">
   <property name="text">
    <string>Rozegraj Mecze</string>
   </property>
  </action>
  <action name="actionStan">
   <property name="text">
    <string>Pokaż stan</string>
   </property>
  </action>
  <action name="actionZaplanuj_spotkania">
   <property name="text">
    <string>Zaplanuj spotkania</string>
   </property>
  </action>
  <action name="actionNastepny_etap">
   <property name="text">
    <string>Przejdz do nastepnego etapu</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
